model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -291,6 +294,24 @@ fun List<ResolvedLicense>.filterExcluded() =
                 resolvedLicense.locations.any { it.matchingPathExcludes.isEmpty() }
     }.map { it.filterExcludedCopyrights() }
 
+/**
+ * Filter licenses for which a resolution exists. Licenses are removed if all the conditions match:
```

Comment:
I find "Filter licenses for which a resolution exists" misleading, because a resolution corresponds to a rule violation, not to a license.